server/controllers/auth/signup.ts

```diff
@@ -11,21 +12,21 @@ import { CustomError } from '../../utils/custom_error';
 export const signupController = async (req: Request, res: Response, next: NextFunction): Promise<any> => {
   try {
     // * Signup data validation.
-    await validateSignup(req.body);
+    const signupData = await validateSignup(req.body);
```

Comment:
this is typed correctly?

Reply:
Now, I'm going to export the schema and validate it here, so it will change.